packages/protocol/lib/backward/ast-code.ts

```diff
@@ -0,0 +1,392 @@
+// tslint:disable: max-classes-per-file
+import { makeZContract } from '@celo/protocol/lib/backward/internal';
+import {
+  BuildArtifacts,
+  Contract as ZContract
+} from '@openzeppelin/upgrades';
+import ContractAST from '@openzeppelin/upgrades/lib/utils/ContractAST';
+
+const VISIBILITY_PUBLIC = 'public'
+const VISIBILITY_EXTERNAL = 'external'
+const CONTRACT_TYPE_CONTRACT = 'contract'
+const STORAGE_DEFAULT = 'default'
+
+const OUT_VOID_PARAMETER_STRING = 'void'
+
+export class ASTCodeCompatibilityReport {
+  changes: Change[];
```

Comment:
The general .ts style in our repo has been to not include semi-colons at the end of a line

Reply:
Yeah I think I was a little bit confused, since I used semi-colons in class definitions but not in general code. I'll fix it